### PR TITLE
 a18-calendar

### PR DESCRIPTION
**Pull request by valentinavargas31**

### CODE REVIEW

**Comment by Iregui44** [comment]
The hours in each line are not alin with each row, colors are a bit too diferent, and format is not the same try and do it more alike the mockup.

File: src/mm/html-css/A18-Calendar/index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    <title>Calendar Dashboard</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="shortcut icon" href="img/favicon.png" type="image/x-icon" />
+  </head>
+  <body>
+    <div class="main-content">
+      <header>
+        <h1>Calendar</h1>
+        <select id="options" name="options">
+          <option value="opcion1">1 All Meetings</option>
```
Comment (Iregui44) [inline src/mm/html-css/A18-Calendar/index.html:15]: All are all metings?

File: src/mm/html-css/A18-Calendar/styles.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
+body {
+  background-color: #ffffff;
+  margin: 0;
```
Comment (Iregui44) [inline src/mm/html-css/A18-Calendar/styles.css:3]: You can add a font for consistency in all the page,

File: src/mm/html-css/A18-Calendar/index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          <option value="opcion2">2 All Meetings</option>
+          <option value="opcion3">3 All Meetings</option>
+        </select>
+        <button class="create-meeting">Create a Meeting</button>
+      </header>
+      <div class="calendar-container">
+        <div class="calendar-nav">
+          <div class="days">
+            <div class=""><button>&lt;</button></div>
+            <div class="day">Sep<br />19<br />Mon</div>
```
Comment (Iregui44) [inline src/mm/html-css/A18-Calendar/index.html:25]: aria-labels?

File: src/mm/html-css/A18-Calendar/styles.css [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.main-content {
+  margin: 2%;
```
Comment (Iregui44) [inline src/mm/html-css/A18-Calendar/styles.css:7]: Add some shados or something to do it more alike the mockup.

File: src/mm/html-css/A18-Calendar/styles.css [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.calendar-body {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+}
+
+.calendar-cell {
+  background: #ffffff;
+  padding: 20px;
```
Comment (Iregui44) [inline src/mm/html-css/A18-Calendar/styles.css:99]: display: flex; may be better for alignment

File: src/mm/html-css/A18-Calendar/index.html (hunk 1 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/index.html:15]: corrected

File: src/mm/html-css/A18-Calendar/styles.css (hunk 2 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/styles.css:3]: ok

File: src/mm/html-css/A18-Calendar/index.html (hunk 3 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/index.html:25]: corrected

File: src/mm/html-css/A18-Calendar/styles.css (hunk 4 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/styles.css:7]: added

File: src/mm/html-css/A18-Calendar/styles.css (hunk 5 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/styles.css:99]: ok

**Comment by Iregui44** [comment]
You havent commit any changes.

**Comment by valentinavargas31** [comment]
realizados los cambios

File: src/mm/html-css/A18-Calendar/index.html (hunk 3 above)
Comment (Iregui44) [reply src/mm/html-css/A18-Calendar/index.html]: I dont see aria-labels for accesibility

File: src/mm/html-css/A18-Calendar/styles.css (hunk 5 above)
Comment (Iregui44) [reply src/mm/html-css/A18-Calendar/styles.css]: Dont see the change

File: src/mm/html-css/A18-Calendar/index.html (hunk 3 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/index.html]: corrected

File: src/mm/html-css/A18-Calendar/styles.css (hunk 5 above)
Comment (valentinavargas31) [reply src/mm/html-css/A18-Calendar/styles.css]: corrected

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**